static/lang/en.json

```diff
@@ -4911,6 +4911,7 @@
             },
             "Date": "{weekday}, {day} of {month}, {year} {era}"
         },
+        "WorldItem": "World Item",
```

Comment:
🥲

Reply:
Other than WorldItemLabel I have no idea what to do with that.